daemon/create.go

```diff
@@ -14,6 +14,16 @@ import (
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
+// ContainerCreateAPI takes configs and creates a container, returning
+// the ID of the newly created container.
+func (daemon *Daemon) ContainerCreateAPI(name string, config *runconfig.Config, hostConfig *runconfig.HostConfig, adjustCPUShares bool) (string, []string, error) {
```

Comment:
Can daemon.ContainerCreate just return the types.ContainerCreateResponse rather than having another function here?